src/plugins/intel_npu/tests/unit/npuw/online_partitioning.cpp

```diff
@@ -0,0 +1,5 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "online_partitioning.hpp"
```

Comment:
please consider more explicit separation to header and c++ files:
c++ file can contain tests, header file can contain test fixture. For your case if no test fixture exists you don't need `online_partitioning.hpp` header, you can safely rename this header to something like `online_partitioning_test_utils.hpp` and may be put it under some `utils/` folder

Reply:
Or another way, you can just take out the generator, or in future, somehow rely on the same one generator in the `functional/` folder (however, I don't know how this can be done)

Reply:
Leaving model generator placing as a separate task. Removed the header